source/server/getting-started/bungeecord.rst

```diff
@@ -36,10 +36,9 @@ connected server, even though the servers are being run in offline mode.
  products, although they are developed by trusted members of the community.
 
 For SpongeVanilla or Vanilla clients connecting to unmodded SpongeForge, IP Forwarding works out of the box.
-`Pull <https://github.com/SpigotMC/BungeeCord/pull/1557>`_
-`requests <https://github.com/SpigotMC/BungeeCord/pull/1678>`_ have been supplied to BungeeCord to allow BungeeCord to
-support SpongeForge natively. We are awaiting it to be included in the main product. For now please use one of the
-solutions mentioned above.
+Sponge supplied `fi <https://github.com/SpigotMC/BungeeCord/pull/1557>`_\
```

Comment:
The fixes were independent of the Sponge team. I did them of my own accord way before I was a member of Sponge, so it is inaccurate to say as such. I'd go with:

"`Pull <https://github.com/SpigotMC/BungeeCord/pull/1557>` `requests <https://github.com/SpigotMC/BungeeCord/pull/1678>` were made to BungeeCord for modded SpongeForge support, but these haven't been accepted. Other solutions (mentioned above) include these patches and should be used instead."

Reply:
As long as the link ReST formatting is correct.

Reply:
Yep, sorry, was writing this from bed last night on my tablet!